sergey.duev/task_5/src/main/java/com/hillel/servlet/BankServlet.java

```diff
@@ -0,0 +1,336 @@
+package com.hillel.servlet;
+
+import com.google.gson.Gson;
+import com.hillel.model.*;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@WebServlet(name = "Bank", urlPatterns = {BankServlet.PATH_BANK_ACCOUNT, BankServlet.PATH_USER})
+public class BankServlet extends HttpServlet {
+
+    private static final String USER_ID = "userId";
+    private static final String BANK_ACCOUNT_ID = "bankAccountId";
+
+    private static final String ERROR_USER_NOT_FOUND = "User not found";
+    private static final String ERROR_BANK_ACCOUNT_NOT_FOUND = "Bank account not found";
+
+    private static final String INFO_USER_CREATED = "User created";
+    private static final String INFO_USER_DELETED = "User deleted";
+    private static final String INFO_USER_UPDATED = "User updated";
+    private static final String INFO_BANK_ACCOUNT_CREATED = "Bank account created";
+    private static final String INFO_BANK_ACCOUNT_DELETED = "Bank account deleted";
+    private static final String INFO_BANK_ACCOUNT_UPDATED = "Bank account updated";
+
+    public static final String PATH_USER = "/bank/user";
+    public static final String PATH_BANK_ACCOUNT = "/bank/bankAccount";
+    private long nextUserId = 1;
+    private long nextBankAccountId = 1;
+    private Gson gson = new Gson();
+
+    private Map<User, List<BankAccount>> bankAccounts = new ConcurrentHashMap<>();
+
+    private boolean checkPath(HttpServletRequest req, String path) {
+        return req.getRequestURI().endsWith(path);
+    }
+
+    private boolean checkPathUser(HttpServletRequest req) {
+        return checkPath(req, PATH_USER);
+    }
+
+    private boolean checkPathBankAccount(HttpServletRequest req) {
+        return checkPath(req, PATH_BANK_ACCOUNT);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
Use different get methods for different servlets

The same for all other methods